TraitsExpanded/GameStatusInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿using TaleWorlds.Core;
+
+namespace TraitsExpanded
+{
+    public class GameStatusInfo
```

Comment:
let's consider making anything we don't want to expose to submodules `internal` rather than `public` visibility

Reply:
I will eventually, but this has to be passed through to trait systems, I will redo the access modifiers a bit later, once I fully get the pipeline working